crates/types/src/invocation.rs

```diff
@@ -198,8 +198,7 @@ pub enum ServiceInvocationResponseSink {
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub enum Source {
     Ingress,
-    // TODO This can be changed to simply InvocationId
-    Service(FullInvocationId),
+    Service(InvocationId),
```

Comment:
Can we also capture the service name? Then we wouldn't have to do the changes on the CLI side. Knowing not only the invocation id but also the service name can be useful information for users.

Reply:
> Would the service name be available at the creation sites of Source::Service in the fid-less world?

No. In my PR about idempotency, i would need to store additional data to reconstruct that, and I don't want to do it, as this diagnostic info can be retrieved anyway differently.

Reply:
Essentially here https://github.com/restatedev/restate/pull/1326/files#diff-3c9cbf4d617758ec9be8fb418fa90278ac98881145eea5a2bced65a2e417978bR75

I would need to store the additional metadata of the source only for this purpose.

Reply:
If we really feel this is a problem, we could re-introduce this later once we sort out all the details of #1329. I also think this is a relative problem of the CLI: in a webui, probably every `invocation_id` will show up as a link to a page showing the invocation status, so the navigation will be much easier in that sense that in CLI.

Reply:
One second i might have overlooked something, i might have a way to preserve this fid.

Reply:
> i might have a way to preserve this fid.

That be great :-)